manifests/pipecd/templates/configmap.yaml

```diff
@@ -200,6 +200,8 @@ data:
             replacement: $1
 
       - job_name: pipecd-ops
+        # This scrape_interval must not be larger than the interval of piped's stats-reporting.
+        # ref: https://github.com/pipe-cd/pipecd/blob/master/pkg/app/piped/statsreporter/reporter.go#L53
```

Comment:
[IMO]
The link leads to the code for the latest master. So, there is a possibility of changing the position if someone change the file (e.g. delete or add some lines)

So how about adding the interval description to the docs and referring to it on this line?

Reply:
Sure, so just finish the PR and later make docs PR 👍

Reply:
> If you worry about the master branch reference, let fix it as snapshot (commit hash) branch reference

@t-kikuc plz change only it 🙏

Reply:
@khanhtc1202 @ffjlabo 
thanks, I fixed using a73d0913643ef1da05bc8898cf055c9893f16e0b (v0.47.0 tag's hash)